test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -407,6 +407,24 @@ public void FastTreeBinaryClassificationTest()
             Done();
         }
 
+        [Fact]
+        [TestCategory("Binary")]
+        [TestCategory("LightGBM")]
+        public void LightGBMClassificationTest()
```

Comment:
>LightGBMClassificationTest [](start = 20, length = 26)

Should we port over more tests? I'm pretty sure we had more than this in the repo we're migrating from.

Reply:
I can try but my concern is some of them might depend on datasets that we may not have in this repo for legal reasons.

---
In reply to: [197461623](https://github.com/dotnet/machinelearning/pull/392#discussion_r197461623) [](ancestors = 197461623)

Reply:
I see. The should we not write tests for things like regression, ranking, multiclass that we currently are using for the other tests that exist in ML.NET?

---
In reply to: [197570510](https://github.com/dotnet/machinelearning/pull/392#discussion_r197570510) [](ancestors = 197570510,197461623)

Reply:
We certainly should and I'm going to try. If the dataset is not available I will try to use an existing one and change the pipeline.

---
In reply to: [197575933](https://github.com/dotnet/machinelearning/pull/392#discussion_r197575933) [](ancestors = 197575933,197570510,197461623)

Reply:
Got everything except ranking for which we need to find a new dataset. I will open an issue and skip the test for ranking. I did swap housing dataset for wine for regression tests.

---
In reply to: [197577162](https://github.com/dotnet/machinelearning/pull/392#discussion_r197577162) [](ancestors = 197577162,197575933,197570510,197461623)